src/widgets/barchart/mod.rs

```diff
@@ -97,66 +102,157 @@ impl<'a> BarChart<'a> {
         self
     }
 
+    /// Surround the [`BarChart`] with a [`Block`].
     pub fn block(mut self, block: Block<'a>) -> BarChart<'a> {
         self.block = Some(block);
         self
     }
 
+    /// Set the value necessary for a [`Bar`] to reach the maximum height.
+    ///
+    /// If not set, the maximum value in the data is taken as reference.
```

Comment:
This might be good info to add to the doc comment for the type.

Reply:
Tried to include it but didn't really see how.
I'm not sure this is really relevant for the majority of workflows. It can still be found while searching the `BarChart` API for advanced usage.
Please let me know if you think otherwise.